syntax = "proto2";

package BackupPB;

message JobResponse{
	enum Status {OK=0; FAIL=1;}
    required string path = 2;
    required Status status = 1;	
    optional string checksum = 3;
}

